content/docs/pipelines/sdk/enviroment_variables.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Using enviroment varibales in piplines"
+description = "How to set and use enviroment variables in kubeflow piplines"
++++
+
+This page describes how to create a enviroment varaibale for Kubeflow Pipelines 
+and how they work in a pipline. 
+
+## Before you start
+
+### Initial setup
+Before starting with this tutorial the following things have to be in place: 
+- [Kubeflow set up on kubernetes](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/)
+- Jupyter nooteboks set up on you kubeflow deployment
+
+
+## Using enviroment variables 
+
+Below is a example python lightweight component where we read a enviroment
```

Comment:
It is a good practice in technical docs not to use "we". I can become confusing for the reader to determine who is acting. More information is available in the [style guide](https://www.kubeflow.org/docs/about/style-guide/#address-the-audience-directly)

Suggested revision:

In this example, you pass an environment variable to a lightweight Python component, which writes the variable's value to the log.